### PR TITLE
KOGITO-3219: Normalize Kogito Travel Basic and Extended examples

### PR DESCRIPTION
**Pull request by Sgitario**
JIRA: https://issues.redhat.com/browse/KOGITO-3219
Description: Rename kogito travel agency examples for basic and extended and use the pom parent surefire plugin version.
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: kogito-travel-agency/basic/pom.xml [hunk 1]
```diff
@@ -7,9 +7,11 @@
     <artifactId>kogito-travel-agency-example</artifactId>
     <version>1.0.0-SNAPSHOT</version>
   </parent>
+  
   <artifactId>kogito-travel-agency-example-basic</artifactId>
+  <name>Kogito Example :: Travel Agency :: Basic</name>
+  
   <properties>
-    <surefire-plugin.version>2.22.0</surefire-plugin.version>
```
Comment (Sgitario) [inline kogito-travel-agency/basic/pom.xml:12]: The surefire version is 2.22.2 which supports parallel builds. Therefore, removing this line will force to use this version from the parent maven.

**Review by sutaakar — APPROVED** [review]

**Comment by ricardozanini** [comment]
@Sgitario @sutaakar can be merged?

**Comment by sutaakar** [comment]
From my point of view yes.

**Comment by Sgitario** [comment]
jenkins please retest this

**Comment by Sgitario** [comment]
> From my point of view yes.

For me, too.

**Pull request merged**